### ISSUE TITLE
prepending origin host to API endpoint

### ISSUE THREAD

**Issue by BernardA** [api-platform/admin#254]
**API Platform version(s) affected**: 1.0.1

**Description**  
It's prepending the origin host to the API endpoint like so:
`Request URL:http://localhost:3000/127.0.0.1:8000/api`

Where `http://localhost:3000` is the origin URL given by `npm-start` and `127.0.0.1:8000.api` is the API endpoint. All locally, of course.

**How to reproduce**  
I am just starting with API-Platform Admin. So I just created a `create-react-app` and changed the `/src/App.js` as instructed. Nothing else.

`/src/App.js

     import React from "react";
     import { HydraAdmin, ResourceGuesser } from "@api-platform/admin";

    export default () => (
      <HydraAdmin entrypoint="http:/127.0.0.1:8000/api">
           <ResourceGuesser name="categories" />
            <ResourceGuesser name="ads" />
      </HydraAdmin>
    );

package.json

    "dependencies": {
      "@api-platform/admin": "^1.0.1",
      "react": "^16.11.0",
      "react-dom": "^16.11.0",
      "react-scripts": "3.2.0"
    },
    "scripts": {
      "start": "react-scripts start",
      "build": "react-scripts build",
      "test": "react-scripts test",
      "eject": "react-scripts eject"
    },
    "eslintConfig": {
      "extends": "react-app"
    },

**Possible Solution**  

**Additional Context**  
It just shows error below on the screen, which is to expect as it is not reaching the endpoint.

   `Error while reading the API schema`

**Comment by BernardA** [comment]
Any ideas on this one? Still getting the wrong URL as per above.
Anyone?

**Comment by BernardA** [comment]
I believe I found where the issue may be.

At `node_modules\@api-platform\admin\lib\hydra\dataProvider.js` or
`node_modules\@api-platform\admin\src\hydra\dataProvider.js` ( I am not sure which one)

        const convertReactAdminRequestToHydraRequest = (type, resource, params) => {
          const entrypointUrl = new URL(entrypoint, window.location.href);
         
With `entrypoint` being `http:/127.0.0.1:8001/api` this will output a URL as `http://localhost:3000/127.0.0.1:8001/api`.

The way its coded it's meant  to generate a URL on the same domain, which I believe is not the intent.

It should just be `const entrypointUrl = new URL(entrypoint);`

I did try to manually change above and rerun the `npm start` but it still produces the wrong URL. There must be somewhere else or there may be a cache issue.
